### DISCUSSION TITLE
does aspect ratio matter for `detect` streams?

### DISCUSSION THREAD

**Discussion by vajonam** [blakeblackshear/frigate#6163] [Ask A Question] [answered]
I have 16:9 camera, full res is 1920x1080, however the substream is the same image squashed to 680x480, at the moment I have the `detect` resolution set to 854x480 (to get a un-squased image) . 

- what is the correct way to adjust in frigate? does it even matter? 854x480 or 640x360? 
- does ffmeg have any degradation in performance when using non 32 or 16 multiple heights and widths? 

record streams the right aspect ratio so everything looks good there.

**Comment by NickM-27** [comment, marked as answer]
You would most likely want the aspect ratio of the detect stream to match the aspect ratio of the camera itself, changing the size of the stream will of course use somewhat more resources
